src/dbnode/encoding/ostream.go

```diff
@@ -251,12 +240,7 @@ func (os *ostream) Reset(buffer checked.Bytes) {
 	}
 }
 
-// Rawbytes returns the Osteam's raw bytes. Note that this does not transfer ownership
-// of the data and bypasses the checked.Bytes accounting so callers should:
-//     1. Only use the returned slice as a "read-only" snapshot of the data in a context
-//        where the caller has at least a read lock on the ostream itself.
-//     2. Use this function with care.
```

Comment:
Hm, can we move these docs somewhere else? They're fairly valuable.

Perhaps to the interface itself?

Reply:
These should be moved to [encoding/types](https://github.com/m3db/m3/pull/2370/files/4af783dd2a56cd7075b3db9c9ecb7d1946aea0b1#diff-2e6541aee73116ef7c93a9618f5063c4R406) to standardize with our other comment style; I can move it back if preferred though

Reply:
Great, no that sounds good.